calculate.py

```diff
@@ -28,14 +30,17 @@ def evaluate(expression: str, is_rad: bool) -> str:
         expression = expression.replace('√', 'sqrt')
 
         # Convert numbers directly to rad
-        if exp_is_only_num(expression):
-            if is_rad == "true":
-                expression = str(display.rad((float)(expression)))
-            elif is_rad == "false":
-                expression = str(display.deg((float)(expression)))
+        # if exp_is_only_num(expression):

```

Comment:
Hey it seems you already commented out the block of code I was thinking of removing. 
Would it be possible to also: 
-remove these lines as well as the "exp_is_only_num" function (unless you're using it)
-remove the onclick for angleMode in html
-fix the type of is_rad in the method signature

Reply:
Just updated this